cmd/container-use/stdio_test.go

```diff
@@ -0,0 +1,359 @@
+package main_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/mark3labs/mcp-go/client"
+	"github.com/mark3labs/mcp-go/mcp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// MCPServerProcess represents a running container-use MCP server
+type MCPServerProcess struct {
+	cmd        *exec.Cmd
+	client     *client.Client
+	repoDir    string
+	configDir  string
```

Comment:
🔧 **Maintainability (Medium):** Build output should be captured for better error diagnostics rather than printing directly to test output

```suggestion
// Capture build output for better test diagnostics
		var stderr, stdout bytes.Buffer
		cmd.Stderr = &stderr
		cmd.Stdout = &stdout
```